<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GlassMaker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-GlassMaker-Desktop_Qt_6_2_0_MinGW_64_bit-Debug/Everaldo-Crystal-Clear-App-miscellaneous.ico</normaloff>../build-GlassMaker-Desktop_Qt_6_2_0_MinGW_64_bit-Debug/Everaldo-Crystal-Clear-App-miscellaneous.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listaSkladnikow">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listaSurowcow">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listaWybranychSkladnikow">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listaWybranychSurowcow">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="przyciskWyboruSkladnikow">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz Skladniki</string>
    </property>
   </widget>
   <widget class="QPushButton" name="przyciskWyboruSurowcow">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz Surowce</string>
    </property>
   </widget>
   <widget class="QPushButton" name="przyciskKolejnegoEtapu">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>520</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Przejdz dalej</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Skladniki</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybrane Skladniki</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Surowce</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>310</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybrane Surowce</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz skladniki oraz surowce, ktore będą obecne w zestawie szklarskim</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>280</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
